stdlib/email/base64mime.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import Callable
+
+def header_length(bytearray: bytes) -> int: ...
+def header_encode(header_bytes: str | bytes, charset: str = ...) -> str: ...
+def body_encode(s: bytes, maxlinelen: int = ..., eol: str = ...) -> str: ...
+def decode(string: str | bytes) -> bytes: ...
```

Comment:
Can apply to both `str` and `bytes`, and clearly has a clause to catch that (despite the misleading `string` param name):
![image](https://user-images.githubusercontent.com/80432516/147124509-18efe049-dd2a-4bf1-8404-9a09a4d99f54.png)